packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -919,16 +919,14 @@ test.serial('onDeadline exit, with chainStorage RPC snapshot', async t => {
   t.is(await E(exitingSeat).getOfferResult(), 'Your bid has been accepted');
   t.false(await E(exitingSeat).hasExited());
 
-  await bookTracker.assertChange({
-    collateralAvailable: { value: 0n },
-  });
+  await bookTracker.assertChange({});
 
   await driver.advanceTo(170n, 'wait');
   await scheduleTracker.assertChange({
     nextDescendingStepTime: { absValue: 175n },
   });
   await bookTracker.assertChange({
-    collateralAvailable: { value: 100n },
```

Comment:
It's not really clear from looking at this test that the problem is fixed. It might help to put comments at various points after `driver.advanceTo` to signify what point in the process the test is at.

In general though at previous orgs I've seen conventions of one unit test per behavior for this reason. Maybe that's unwieldy for these scenarios but reading the test name `onDeadline exit, with chainStorage RPC snapshot`, doesn't tell me anything about the expected behavior.